docs/protocol/core/v3.0.rst

```diff
@@ -665,7 +802,7 @@ ignored if not understood::
 
     {
         "zarr_format": "https://purl.org/zarr/spec/protocol/core/3.0",
-        "metadata_encoding": "application/json",
+        "metadata_encoding": "https://tools.ietf.org/html/rfc8259",
         "extensions": [
```

Comment:
let's go with `https://purl.org/zarr/spec/protocol/core/3.0` + anchor to metadata section.